splinepy/bspline.py

```diff
@@ -122,6 +122,75 @@ def insert_knots(self, parametric_dimension, knots):
 
         return inserted
 
+    def uniform_refine(
+        self, parametric_dimensions=None, degree_of_refinement=1
```

Comment:
```suggestion
        self, para_dim=None, n_knots=1
```
or  something like `n_knots_per_knot_span`